javascript/ql/src/semmle/javascript/security/dataflow/ClientSideUrlRedirectCustomizations.qll

```diff
@@ -145,4 +145,17 @@ module ClientSideUrlRedirect {
       )
     }
   }
+  
+  /**
+   * A write of an attribute which may execute JavaScript code or 
```

Comment:
```suggestion

  /**
   * A write of an attribute which may execute JavaScript code or
```

Our autoformatter doesn't like trailing whitespace. With this change all our CI checks should pass.